frontend/src/components/ProjectMemberTable.vue

```diff
@@ -175,6 +181,18 @@ export default {
         const ownerList: ProjectMember[] = [];
         const developerList: ProjectMember[] = [];
         for (const member of props.project.memberList) {
+          if (
+            !props.activeRoleProvider &&
+            member.roleProvider !== props.project.roleProvider
```

Comment:
would be more readable to use

member.roleProvider == props.activeRoleProvider

?